### PR TITLE
Idea card skeleton

### PR DESCRIPTION
**Pull request by vedmant**
#498

### CODE REVIEW

File: front/components/ideaDetail/IdeaCardSkeleton.vue [hunk 1]
```diff
@@ -0,0 +1,98 @@
+<template>
+  <div class="idea-card-skeleton">
```
Comment (aldarund) [inline front/components/ideaDetail/IdeaCardSkeleton.vue:2]: maybe we can use https://vuetifyjs.com/en/components/skeleton-loaders/ ?

Comment (vedmant) [reply front/components/ideaDetail/IdeaCardSkeleton.vue:2]: Yes, didn't know that it has this components, I'll redo

Comment (vedmant) [reply front/components/ideaDetail/IdeaCardSkeleton.vue:2]: Done

**Comment by ezl** [comment]
merging, @aldarund can't access github but he let me know it can be merged.

**Pull request merged**